sandbox/lib/presets.dart

```diff
@@ -16,6 +16,9 @@ class Presets {
       _flee1(),
       _flee2(),
     ]),
+    _PresetGroup('Seperate', [
```

Comment:
Could you please rename `Seperate` -> `Separate` everywhere? My spell checker complains :)

Reply:
Ok I will do it now
After that I need to push pr in this branch or need to create a new branch ??

Reply:
You should be able to commit in your local branch, and then push directly, and that would update the current PR. So, no need for new PRs :)